src/API/Site/Controllers/Ads/AssetGroupController.php

```diff
@@ -121,7 +127,7 @@ public function edit_asset_group_params(): array {
 	 */
 	public function get_asset_group_params(): array {
 		return [
-			'id' => [
+			'campaign_id' => [
```

Comment:
💅   I guess we should make this param required? 🤔

Reply:
Otherwise I got a fatal when running the endpoint without `campaing_id` param.

```
 2023/01/16 09:57:51 [error] 30#30: *4801 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught TypeError: Argument 1 passed to Automattic\WooCommerce\GoogleListingsAndAds\API\Google\AdsAssetGroup::get_asset_groups_by_campaign_id() must be of the type int, null given, called in /var/www/html/wp-content/plugins/google-listings-and-ads/src/API/Site/Controllers/Ads/AssetGroupController.php on line 153 and defined in /var/www/html/wp-content/plugins/google-listings-and-ads/src/API/Google/AdsAssetGroup.php:273
```